pkg/appfile/service.go

```diff
@@ -114,13 +114,14 @@ func (s Service) RenderService(tm template.Manager, name, ns string, cg configGe
 	// render traits
 	traits := make([]v1alpha2.ComponentTrait, 0)
 	for traitType, traitData := range traitKeys {
-		ts, err := evalTraits(tm.LoadTemplate(traitType), ctxData, intifyValues(traitData))
+		defName, rawTmpl := tm.LoadTemplate(traitType)
```

Comment:
we should change all the traitType to use definition name instead of magic Variable from template